### DISCUSSION TITLE
CMAKE = none when trying to run scrollable or any example project

### DISCUSSION THREAD

**Discussion by cole-kitroser** [hecrj/iced#1215] [Q&A]
I've working on trying to understand the iced library for rust and whenever I try to run "cargo run --package scrollable", I am met with an error about how cargo is unable to connect to cmake. I don't know if this is just a rust problem or an iced problem, but any help would be appreciated. I included the full error in the post. My cmake is on version 3.22.0 and rust and cargo are both on version 1.56.1. Any help would be greatly appreciated. Thank you in advance.

[error.txt](https://github.com/iced-rs/iced/files/7620985/error.txt)

**Comment by hecrj** [comment]
This is most likely an issue with your setup, since all of the `iced` codebase is continuously built and tested on GitHub CI for every commit to the repository.

I recommend you to [join the `iced` Discord server](https://discord.gg/3xZJ65GAhd) and ask for help there, since debugging setups asynchronously is not ideal.
